lagom-client-restaurant/menu-item-impl/deploy/kubernetes/menu-item-config.yaml

```diff
@@ -3,7 +3,5 @@ kind: ConfigMap
 metadata:
   name: menu-item-config
 data:
-  CASSANDRA_CONTACT_POINT: "cassandra.cassandra"
-  KAFKA_BROKERS_SERVICE_URL: "restaurant-strimzi-kafka-bootstrap.kafka:9092"
-  #ALLOWED_HOST: "menu-item-svc.default"
-  ALLOWED_HOST: "."
\ No newline at end of file
+  CASSANDRA_SERVICE_NAME: "_cql._tcp.cassandra.cassandra"
+  KAFKA_SERVICE_NAME: "_clients._tcp.restaurant-strimzi-kafka-bootstrap.kafka"
```

Comment:
This now uses the built-in service discovery functionality in Lagom, on top of DNS SRV provided by Kubernetes, to find the Cassandra contact points and Kafka bootstrap servers. These environment variable names are already pre-configured in Lagom to allow you to pass in the service lookup names for Cassandra and Kafka.